src/sr/templates.js

```diff
@@ -302,7 +292,7 @@ class MeasurementStatisticalProperties extends Template {
                 value: options.authority,
                 relationshipType: RelationshipTypes.HAS_PROPERTIES
             });
-            this.push(authorityItem);
+            this.push(descriptionItem);
```

Comment:
Looks like this was a bug before and isn't part of the code formatting?

Reply:
True but I catched it through lint

Reply:
Cool!  Good catch 👍